demos/serverless/deploy.js

```diff
@@ -192,39 +191,50 @@ function createCaptureS3Buckets(bucketPrefix, regions) {
     }]
   });
   fs.writeFileSync('build/lifecycle_configuration.json', lifecycleConfiguration, {encoding: 'utf8', flag: 'w'});
-  for (bucketRegion of regions) {
-    const bucketName = `${bucketPrefix}-${bucketRegion}`;
-    const bucketPolicy = JSON.stringify({
-      "Id": "Policy1625687208360",
-      "Version": "2012-10-17",
-      "Statement": [{
-        "Sid": "Stmt1625687206729",
-        "Action": [
-          "s3:PutObject",
-          "s3:PutObjectAcl"
-        ],
-        "Effect": "Allow",
-        "Resource": `arn:aws:s3:::${bucketName}/*`,
-        "Principal": {
-          "Service": [
-            chimeServicePrincipal
-          ]
-        }
-      }]
-    });
-
-    fs.writeFileSync('build/bucket_policy.json', bucketPolicy, {encoding: 'utf8', flag: 'w'});
+  if (JSON.parse(useChimeSDKMeetings)) {
+    console.log(`Creating S3 buckets for media capture pipelines artifacts.  Bucket prefix: ${bucketPrefix} Regions:[${region}, "us-east-1"]`);
+    createS3Bucket(bucketPrefix, region);
+    createS3Bucket(bucketPrefix, "us-east-1");
```

Comment:
Instead of hard-coding, can we just pass in us-east-1 as a command parameter?

Reply:
I think instead of passing as a parameter, we can create a constant - defaultMediaCaptureRegion= "us-east-1"

Reply:
But why does it has to be us-east-1 though ?
>Create media capture S3 buckets only in us-east-1 and in the respective region, instead of creating in all 14 regions. We need to test only in these two for new ChimeSDK meetings.

This can't be the reason. This is a demo code and we should refrain from editing the code for a specific testing purpose.

Reply:
Creating bucket in us-east-1 region too because Gamma Chime SDK meetings can not be placed in other regions than us-east-1. So, if we want to run media capture tests with Demo app Urls pointing to Gamma Chime SDK meetings, we need to place meeting in us-east-1 and create media capture bucket in us-east-1 region.

Reply:
All of that seems like can be controls through commands? From an external perspective, there is no such thing as a gamma endpoint.

Reply:
Yes, right. Good point. From external perspective, we should be able to start media capture in any media region. I will revert the code.